octopusdeploy_framework/resource_environment.go

```diff
@@ -19,6 +19,8 @@ type environmentTypeResource struct {
 	*Config
 }
 
+var environmentSchema = schemas.EnvironmentSchema{}
```

Comment:
Why did you change the convention used in earlier classes where you straight assign to `resp.Schema` without a local parameter?

Reply:
Not sure, I think this was one of the earlier changes I made. I will go through and make this consistent

Reply:
fixed